webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
+	volumeName, isExist := backup.Labels[types.LonghornLabelBackupVolume]
+	if !isExist {
+		err := errors.Wrap(err, "can not find the backup volume label")
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volume, err := b.ds.GetVolumeRO(volumeName)
+	if err != nil {
+		err := errors.Wrapf(err, "failed to get volume: %v", volumeName)
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volumeAccessMode := longhorn.AccessModeReadWriteOnce
+	if volume.Spec.AccessMode != "" {
+		volumeAccessMode = volume.Spec.AccessMode
+	}
+	backupLabels[types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode)] = string(volumeAccessMode)
+
+	kubeStatus, err := json.Marshal(volume.Status.KubernetesStatus)
+	if err != nil {
+		return nil, werror.NewInvalidError(errors.Wrapf(err, "failed to convert k8s status as JSON string").Error(), "")
+	}
+	backupLabels[types.KubernetesStatusLabel] = string(kubeStatus)
```

Comment:
What is the purpose of adding the `KubernetesStatusLabel` label here?

Reply:
Since it saves the kubernetes status into backup through API, so the `KubernetesStatusLabel` should also be handled in mutating to avoid overwriting. 

https://github.com/longhorn/longhorn-manager/blob/127a37f605b13e133a547dfc89470cf3670e2236/api/snapshot.go#L177-L183

Reply:
Yeah, that's why I mentioned the label overriding issue in https://github.com/longhorn/longhorn-manager/pull/1486#discussion_r962089562

Reply:
Since modify the `backupLabels` to avoid overwriting existing value, therefore, I also remove insert Kubernetes status here.